src/vcpkg/commands.init-registry.cpp

```diff
@@ -0,0 +1,31 @@
+#include <vcpkg/base/files.h>
+#include <vcpkg/base/system.print.h>
+
+#include <vcpkg/commands.init-registry.h>
+
+namespace vcpkg::Commands::InitRegistry
+{
+    void perform_and_exit(const VcpkgCmdArguments&, Files::Filesystem& fs)
+    {
+        const auto ports = fs::u8path("ports");
+        const auto baseline = fs::u8path("versions") / fs::u8path("baseline.json");
+        if (!fs.exists(ports))
+        {
+            fs.create_directory(ports, VCPKG_LINE_INFO);
+        }
+        if (!fs.exists(baseline))
+        {
+            const auto content = R"({
+  "default": {}
+})";
+            fs.write_contents_and_dirs(baseline, content, VCPKG_LINE_INFO);
+        }
+        System::print2("Sucessfully created registry at ", fs::u8string(fs.current_path(VCPKG_LINE_INFO)), "\n");
+        Checks::exit_success(VCPKG_LINE_INFO);
```

Comment:
This should take a path.

Reply:
So that you can specify where the registry should be created?

Reply:
Yeah, I think it's very likely new users will not expect it to "just do it" in the current directory. It would be great if `vcpkg init-registry` would print help, and then users could use `vcpkg init-registry .` to "just do it" in the cwd.

Reply:
> I think it's very likely new users will not expect it to "just do it" in the current directory

You only know the command when you have read the usage text. And the usage text says `Initializes a registry in the current directory`. I will add a path parameter, but imho `vcpkg init-registry` should continue creating the registry in the cwd, because you have read the usage and decided against passing a path (and then it has the same behavior as `git init` )

Reply:
Unfortunately, users often don't read docs (or worse -- they skim docs and copy paste commands with wild abandon).